src/sys/windows/stdnet/listener.rs

```diff
@@ -0,0 +1,83 @@
+use std::os::windows::io::{AsRawSocket, FromRawSocket, IntoRawSocket, RawSocket};
+use std::{fmt, io, mem};
+
+use windows_sys::Win32::Networking::WinSock::SOCKET_ERROR;
+
+use super::{socket::Socket, SocketAddr};
+
+pub(crate) struct UnixListener(Socket);
+
+impl fmt::Debug for UnixListener {
```

Comment:
Can move this below the impls, following the same conventions as the rest of the code base.